pyttb/tensor.py

```diff
@@ -1796,6 +2238,16 @@ def __rtruediv__(self, other):
         Returns
         -------
         :class:`pyttb.tensor`
+
+        Examples
+        --------
+        >>> T = ttb.tensor(np.array([[1, 2], [3, 4]]))
+        >>> np.set_printoptions(precision=8)
```

Comment:
An alternative is to use the `...` doctest description to avoid comparing the full float. I think we use that a variety of other places, for example nvecs

Reply:
That did not work in all places, but I can try again. I am using Github codespaces (while traveling) and there seem to be some differences from my typically VS code setup.